typescript2/app-promptfiddle/vercel-build.sh

```
@@ -0,0 +1,68 @@
+dnf install -y git wget tar gzip bzip2 xz
+
+if ! command -v curl &> /dev/null; then
+    echo "Error: curl is not available"
+    exit 1
+fi
+
+# --- Rust toolchain ----------------------------------------------------------
+echo "Installing Rust..."
+if ! command -v rustc &> /dev/null; then
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable
+    source "$HOME/.cargo/env"
```

Shellcheck (0.11.0)

[info] Not following: ./.cargo/env was not specified as input (see shellcheck -x).

(SC1091)